pom.xml

```diff
@@ -368,16 +374,16 @@
       <version>1.2.6.RELEASE</version>
     </dependency>
 
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-databind</artifactId>
-      <version>${jackson.version}</version>
-    </dependency>
-
     <dependency>
       <groupId>com.sendgrid</groupId>
       <artifactId>sendgrid-java</artifactId>
       <version>4.3.0</version>
+      <exclusions>
+          <exclusion>
+              <groupId>fasterxml.jackson.core</groupId>
```

Comment:
I think this should be **_com_**.fasterxml...

Reply:
Ugh, yes